app/web/src/store/realtime/realtime.store.ts

```diff
@@ -183,6 +183,28 @@ export const useRealtimeStore = defineStore("realtime", () => {
     });
   }
 
+  const sendCursor = (req: {
```

Comment:
I might abstract this so the realtime store just knows about how to send a message (and handle json stuff) and then something like the "presence store" would deal with handling all the incoming and outgoing stuff about presence

Reply:
Yeah, we definitely will want to centralize that, but again. Let's wait until other real time events are added. That will show us how to best encapsulate while supporting other cases.

Doesn't feel required for this first iteration.